src/rgw/rgw_process.cc

```diff
@@ -320,6 +320,10 @@ int process_request(rgw::sal::RGWRadosStore* const store,
 	  << " op status=" << op_ret
 	  << " http_status=" << s->err.http_ret
 	  << " latency=" << s->time_elapsed()
+	  << " op_method=" << s->info.method
+	  << " op_target=" << s->info.request_uri
+	  << " op_content_length=" << s->content_length
```

Comment:
```suggestion
	  << " op_obj_size=" << s->obj_size
```
`s->content_length` was working only on PUT,  `s->obj_size` shows value on both PUT and GET